build-image/first-boot.sh

```diff
@@ -57,11 +61,8 @@ elif grep -q "openHABian" /etc/wpa_supplicant/wpa_supplicant.conf; then
     rfkill unblock wifi
     ifconfig wlan0 up
     if iwlist wlan0 scanning 2>&1 | grep -q "Interface doesn't support scanning"; then
-      echo "FAILED"
-      echo ""
-      echo "I was not able to turn on the wifi"
-      echo "Here is some more information"
-      echo ""
+      echo "FAILED" 
+      echo -n "$(timestamp) [openHABian] I was not able to turn on the wifi \n Here is some more information: \n"
```

Comment:
Everywhere you wrote " \n Here" you'll have a whitespace at the beginning of the line. I'd suggest to fix that.

Reply:
Will fix!